scalalib/src/mill/scalalib/TestModuleUtil.scala

```diff
@@ -355,16 +362,39 @@ private final class TestModuleUtil(
       }
     }
 
+    val filteredClassCount = filteredClassLists.map(_.size).sum
     val executor = Executors.newScheduledThreadPool(1)
     val outputs =
       try {
-        // Periodically check the queueLog file of every runner, and tick the executing test name
+        // Periodically check the claim log and claim stats files of every runner, and tick the relevant infos
         executor.scheduleWithFixedDelay(
-          () =>
+          () => {
+            // reuse to reduce syscall, may not be as accurate as running `System.currentTimeMillis()` inside each entry
+            val now = System.currentTimeMillis()
             workerStatusMap.forEach { (claimLog, callback) =>
               // the last one is always the latest
-              os.read.lines(claimLog).lastOption.foreach(callback)
-            },
+              os.read.lines(claimLog).lastOption.foreach { currentTestClass =>
+                testClassTimeMap.putIfAbsent(currentTestClass, now)
+                val last = testClassTimeMap.get(currentTestClass)
+                val suffix = ((now - last) / 1000).toInt match {
+                  case 0 => ""
+                  case n => s" ${n}s"
+                }
```

Comment:
Let's move `PromptLoggerUtil.renderSecondsSuffix` to `mill.internal.Util` so we can re-use it here